### DISCUSSION TITLE
support for videos?

### DISCUSSION THREAD

**Discussion by nrbnlulu** [slint-ui/slint#2299] [Q&A] [answered]
I need to show video coming from rtsp camera is this supported?

**Comment by tronical** [comment, marked as answer]
We're sadly behind schedule with #977, but that ticket is basically the infrastructure we think would be one for for Slint, so that you can use third-party multimedia framework to decode audio and video and feed the video into Slint.

**Reply by nrbnlulu** [reply]
Thanks for the reply.  
This project seems very promising!
I will definitely migrate from QML when this project gets mature enough for my use case.
